### PR TITLE
Add ID translation for define-command-argument-container

### PR DESCRIPTION
**Pull request by adityaalifn**
add ID translation for `/docs/tasks/inject-data-application/define-command-argument-container/`

related to this issue: https://github.com/kubernetes/website/issues/20111

/language id
/assign @kubernetes/sig-docs-id-reviews

### CODE REVIEW

File: content/id/docs/tasks/inject-data-application/define-command-argument-container.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+
+{{% /capture %}}
+
+
+{{% capture steps %}}
+
+## Mendefinisikan sebuah perintah dan argument-argumen saat kamu membuat sebuah Pod
+
+Saat kamu membuat sebuah Pod, kamu dapat mendefinisikan sebuah perintah dan argumen-argumen untuk
+kontainer-kontainer yang berjalan di dalam Pod. Untuk mendefinisikan sebuah perintah, sertakan
```
Comment (girikuncoro) [inline content/id/docs/tasks/inject-data-application/define-command-argument-container.md:28]:
```suggestion
Container-Container yang berjalan di dalam Pod. Untuk mendefinisikan sebuah perintah, sertakan
```

File: content/id/docs/tasks/inject-data-application/define-command-argument-container.md [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+bidang `command` di dalam berkas konfigurasi. Untuk mendefinisikan argumen-argumen untuk perintah, sertakan
+bidang `args` di berkas konfigurasi. Perintah dan argumen-argumen yang telah
```
Comment (girikuncoro) [inline content/id/docs/tasks/inject-data-application/define-command-argument-container.md:30]: for consistency with previous sentence
```suggestion
bidang `args` di dalam berkas konfigurasi. Perintah dan argumen-argumen yang telah
```

File: content/id/docs/tasks/inject-data-application/define-command-argument-container.md [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+kamu definisikan tidak dapat diganti setelah Pod terbuat.
```
Comment (girikuncoro) [inline content/id/docs/tasks/inject-data-application/define-command-argument-container.md:31]:
```suggestion
kamu definisikan tidak dapat diganti setelah Pod telah terbuat.
```

File: content/id/docs/tasks/inject-data-application/define-command-argument-container.md [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+
+Perintah dan argumen-argumen yang kamu definisikan di berkas konfigurasi
```
Comment (girikuncoro) [inline content/id/docs/tasks/inject-data-application/define-command-argument-container.md:33]: nit
```suggestion
Perintah dan argumen-argumen yang kamu definisikan di dalam berkas konfigurasi
```

File: content/id/docs/tasks/inject-data-application/define-command-argument-container.md [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+membatalkan perintah dan argumen-argumen standar (default) yang disediakan oleh _image_ kontainer.
```
Comment (girikuncoro) [inline content/id/docs/tasks/inject-data-application/define-command-argument-container.md:34]: "default" translates to "bawaan"
```suggestion
membatalkan perintah dan argumen-argumen bawaan yang disediakan oleh _image_ Container.
```

File: content/id/docs/tasks/inject-data-application/define-command-argument-container.md [hunk 6]
```diff
@@ -0,0 +1,156 @@
...
+Jika kamu mendefinisikan argumen-argumen, tetapi tidak mendefinisikan sebuah perintah, perintah standar digunakan
```
Comment (girikuncoro) [inline content/id/docs/tasks/inject-data-application/define-command-argument-container.md:35]:
```suggestion
Jika kamu mendefinisikan argumen-argumen, tetapi tidak mendefinisikan sebuah perintah, perintah bawaan digunakan
```

File: content/id/docs/tasks/inject-data-application/define-command-argument-container.md [hunk 7]
```diff
@@ -0,0 +1,156 @@
...
+dengan argumen-argumen baru kamu.
+
+{{< note >}}
+Bidang `command` menyerupai `entrypoint` di beberapa _runtime_ kontainer.
```
Comment (girikuncoro) [inline content/id/docs/tasks/inject-data-application/define-command-argument-container.md:39]:
```suggestion
Bidang `command` menyerupai `entrypoint` di beberapa _runtime_ Container.
```

File: content/id/docs/tasks/inject-data-application/define-command-argument-container.md [hunk 8]
```diff
@@ -0,0 +1,156 @@
...
+Merujuk pada [catatan](#catatan) di bawah.
+{{< /note >}}
+
+Di latihan ini, kamu akan membuat sebuah Pod baru yang menjalankan sebuah kontainer. Berkas konfigurasi
```
Comment (girikuncoro) [inline content/id/docs/tasks/inject-data-application/define-command-argument-container.md:43]: nit
```suggestion
Pada latihan ini, kamu akan membuat sebuah Pod baru yang menjalankan sebuah Container. Berkas konfigurasi
```

File: content/id/docs/tasks/inject-data-application/define-command-argument-container.md [hunk 9]
```diff
@@ -0,0 +1,156 @@
...
+    kubectl apply -f https://k8s.io/examples/pods/commands.yaml
+    ```
+
+2. Daftar Pod yang sedang berjalan
+
+    ```shell
+    kubectl get pods
+    ```
+
+    Keluaran menunjukkan bahwa kontainer yang berjalan di Pod command-demo
```
Comment (girikuncoro) [inline content/id/docs/tasks/inject-data-application/define-command-argument-container.md:60]:
```suggestion
    Keluaran menunjukkan bahwa Container yang berjalan di dalam Pod command-demo
```

File: content/id/docs/tasks/inject-data-application/define-command-argument-container.md [hunk 10]
```diff
@@ -0,0 +1,156 @@
...
+    telah selesai.
+
+3. Untuk melihat keluaran dari perintah yang berjalan di kontainer, lihat _log_
```
Comment (girikuncoro) [inline content/id/docs/tasks/inject-data-application/define-command-argument-container.md:63]: log is indonesian word https://kbbi.web.id/log
```suggestion
3. Untuk melihat keluaran dari perintah yang berjalan di dalam Container, lihat log
```

File: content/id/docs/tasks/inject-data-application/define-command-argument-container.md [hunk 11]
```diff
@@ -0,0 +1,156 @@
...
+env:
+- name: MESSAGE
+  value: "hello world"
+command: ["/bin/echo"]
+args: ["$(MESSAGE)"]
+```
+
+Ini berarti kamu dapat mendefinisikan sebuah argumen untuk sebuah Pod menggunakan
+salah satu teknik yang tersedia untuk mendefinisikan variabel-variabel lingkungan, termasuk
+[ConfigMaps](/docs/tasks/configure-pod-container/configure-pod-configmap/)
```
Comment (girikuncoro) [inline content/id/docs/tasks/inject-data-application/define-command-argument-container.md:93]: singular is preferred, and use the indonesian link
```suggestion
[ConfigMap](/id/docs/tasks/configure-pod-container/configure-pod-configmap/)
```

File: content/id/docs/tasks/inject-data-application/define-command-argument-container.md [hunk 12]
```diff
@@ -0,0 +1,156 @@
...
+dan
+[Secrets](/docs/concepts/configuration/secret/).
```
Comment (girikuncoro) [inline content/id/docs/tasks/inject-data-application/define-command-argument-container.md:95]: singular is preffered, also use the indonesian link
```suggestion
[Secret](/id/docs/concepts/configuration/secret/).
```

File: content/id/docs/tasks/inject-data-application/define-command-argument-container.md [hunk 13]
```diff
@@ -0,0 +1,156 @@
...
+
+{{< note >}}
+Variabel lingkugan muncul dalam tanda kurung, `"$(VAR)"`. Ini
+dibutuhkan untuk variabel yang akan diperuluas di bidang `command` atau `args`.
+{{< /note >}}
+
+## Menjalankan sebuah perintah di sebuah shell
```
Comment (girikuncoro) [inline content/id/docs/tasks/inject-data-application/define-command-argument-container.md:102]: nit
```suggestion
## Menjalankan sebuah perintah di dalam shell
```

File: content/id/docs/tasks/inject-data-application/define-command-argument-container.md [hunk 14]
```diff
@@ -0,0 +1,156 @@
...
+
+Di beberapa kasus, kamu butuh perintah untuk menjalankan sebuah _shell_. Contohnya, 
+perintah kamu mungkin terdiri dari beberapa perintah yang disatukan, atau  mungkin berupa
```
Comment (girikuncoro) [inline content/id/docs/tasks/inject-data-application/define-command-argument-container.md:105]: digabungkan might be better, but it's your choice
```suggestion
perintah kamu mungkin terdiri dari beberapa perintah yang digabungkan, atau  mungkin berupa
```

File: content/id/docs/tasks/inject-data-application/define-command-argument-container.md [hunk 15]
```diff
@@ -0,0 +1,156 @@
...
+args: ["-c", "while true; do echo hello; sleep 10;done"]
+```
+
+## Catatan
+
+Tabel ini merangkum nama-nama bidang yang digunakan oleh Docker dan Kubernetes.
+
+|              Deskripsi                    |    Nama bidang pada Docker   | Nama bidang pada Kubernetes |
+|-------------------------------------------|------------------------------|-----------------------------|
+|  Perintah yang dijalankan oleh kontainer  |         Entrypoint           |            command          |
```
Comment (girikuncoro) [inline content/id/docs/tasks/inject-data-application/define-command-argument-container.md:119]:
```suggestion
|  Perintah yang dijalankan oleh Container  |         Entrypoint           |            command          |
```

File: content/id/docs/tasks/inject-data-application/define-command-argument-container.md [hunk 16]
```diff
@@ -0,0 +1,156 @@
...
+|  Argumen diteruskan ke perintah           |            Cmd               |             args            |
+
+Saat kamu mengesampingkan Entrypoint dan Cmd standar, 
+When you override the default Entrypoint and Cmd, aturan-aturan ini berlaku:
```
Comment (girikuncoro) [inline content/id/docs/tasks/inject-data-application/define-command-argument-container.md:123]: left over english
```suggestion
aturan-aturan ini berlaku:
```

File: content/id/docs/tasks/inject-data-application/define-command-argument-container.md [hunk 17]
```diff
@@ -0,0 +1,156 @@
...
+
+* Jika kamu tidak menyediakan `command` atau `args` untuk sebuah Kontainer,
```
Comment (girikuncoro) [inline content/id/docs/tasks/inject-data-application/define-command-argument-container.md:125]:
```suggestion
* Jika kamu tidak menyediakan `command` atau `args` untuk sebuah Container,
```

File: content/id/docs/tasks/inject-data-application/define-command-argument-container.md [hunk 18]
```diff
@@ -0,0 +1,156 @@
...
+akan digunakan yang telah didefinisikan di _image_ Docker.
```
Comment (girikuncoro) [inline content/id/docs/tasks/inject-data-application/define-command-argument-container.md:126]: nit
```suggestion
maka `command` dan `args` yang didefinisikan di dalam _image_ Docker akan digunakan.
```

File: content/id/docs/tasks/inject-data-application/define-command-argument-container.md [hunk 19]
```diff
@@ -0,0 +1,156 @@
...
+
+* Jika kamu menyediakan `command` tetapi tidak menyediakan `args` untuk sebuah kontainer, akan digunakan
```
Comment (girikuncoro) [inline content/id/docs/tasks/inject-data-application/define-command-argument-container.md:128]:
```suggestion
* Jika kamu menyediakan `command` tetapi tidak menyediakan `args` untuk sebuah Container, akan digunakan
```

File: content/id/docs/tasks/inject-data-application/define-command-argument-container.md [hunk 20]
```diff
@@ -0,0 +1,156 @@
...
+`command` yang disediakan. Entrypoint dan Cmd standar yang didefinisikan dalam 
+_image_ Docker diabaikan.
+
+* Jika kamu hanya menyediakan `args` untuk sebuah kontainer, Entrypoint standar yang didefinisikan di
```
Comment (girikuncoro) [inline content/id/docs/tasks/inject-data-application/define-command-argument-container.md:132]:
```suggestion
* Jika kamu hanya menyediakan `args` untuk sebuah Container, Entrypoint bawaan yang didefinisikan di dalam
```

File: content/id/docs/tasks/inject-data-application/define-command-argument-container.md [hunk 21]
```diff
@@ -0,0 +1,156 @@
...
+_image_ Docker dijalakan dengan `args` yang kamu sediakan.
+
+* Jika kamu menyediakan `command` dan `args`, Entrypoint dan Cmd standar yang didefinisikan
+di _image_ Docker diabaikan. `command` kamu akan dijalankan dengan `args` kamu.
```
Comment (girikuncoro) [inline content/id/docs/tasks/inject-data-application/define-command-argument-container.md:136]: nit
```suggestion
di dalam _image_ Docker diabaikan. `command` kamu akan dijalankan dengan `args` kamu.
```

File: content/id/docs/tasks/inject-data-application/define-command-argument-container.md [hunk 22]
```diff
@@ -0,0 +1,156 @@
...
+|     `[/ep-1]`      |   `[foo bar]`    |      `[/ep-2]`      |   &lt;not set&gt;  |     `[ep-2]`     |
+|     `[/ep-1]`      |   `[foo bar]`    |   &lt;not set&gt;   |     `[zoo boo]`    | `[ep-1 zoo boo]` |
+|     `[/ep-1]`      |   `[foo bar]`    |   `[/ep-2]`         |     `[zoo boo]`    | `[ep-2 zoo boo]` |
+
+
+{{% /capture %}}
+
+{{% capture whatsnext %}}
+
+* Pelajari lebih lanjut tentang [menkonfigurasi Pod and kontainer](/docs/tasks/).
```
Comment (girikuncoro) [inline content/id/docs/tasks/inject-data-application/define-command-argument-container.md:152]: use indonesian link
```suggestion
* Pelajari lebih lanjut tentang [mengatur Pod and Container](/id/docs/tasks/).
```

File: content/id/docs/tasks/inject-data-application/define-command-argument-container.md [hunk 23]
```diff
@@ -0,0 +1,156 @@
...
+* Pelajari lebih lanjut tentang [menjalankan perintah dalam sebuah kontainer](/docs/tasks/debug-application-cluster/get-shell-running-container/).
```
Comment (girikuncoro) [inline content/id/docs/tasks/inject-data-application/define-command-argument-container.md:153]: use indonesian link
```suggestion
* Pelajari lebih lanjut tentang [menjalankan perintah di dalam sebuah Container](/id/docs/tasks/debug-application-cluster/get-shell-running-container/).
```

File: content/id/docs/tasks/inject-data-application/define-command-argument-container.md [hunk 24]
```diff
@@ -0,0 +1,156 @@
...
+* Lihat [Kontainer](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#container-v1-core).
```
Comment (girikuncoro) [inline content/id/docs/tasks/inject-data-application/define-command-argument-container.md:154]:
```suggestion
* Lihat [Container](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#container-v1-core).
```

File: content/id/docs/tasks/inject-data-application/define-command-argument-container.md [hunk 25]
```diff
@@ -0,0 +1,156 @@
+---
+title: Mendefinisikan Perintah dan Argumen untuk sebuah Kontainer
```
[the remark this answers is not in the document]
Comment (adityaalifn) [reply content/id/docs/tasks/inject-data-application/define-command-argument-container.md:2]: okay mas. will start to use `Container` in the future

**Comment by girikuncoro** [comment]
Thanks @adityaalifn looking great! I've also cleared up some pages that were claimed before but not response in milestone issue tracker, feel free if you want to take some

/lgtm
/approve

**Pull request merged**